clang/lib/DPCT/RulesAsm/Parser/AsmTokenKinds.def

```diff
@@ -12,7 +12,7 @@
 #endif
 
 #ifndef PUNCTUATOR
-#define PUNCTUATOR(X,Y) TOK(X)
+#define PUNCTUATOR(X, Y) TOK(X)
```

Comment:
Only format changes, can do it in a separate PR.

Reply:
Okay, I'm reverting this change back.